handsontable/src/plugins/manualRowResize/__tests__/manualRowResize.spec.js

```diff
@@ -1622,4 +1622,69 @@ describe('manualRowResize', () => {
       expect(desiredHeightsLog).toEqual([$rowHeader.height() + 1 - 50 + 6]);
     });
   });
+
+  describe('Test with zoom level 125%', () => {
+    beforeAll(() => {
+      document.body.style.zoom = '1.25';
+    });
+
+    afterAll(() => {
+      document.body.style.zoom = '1';
+    });
+
+    it('should properly set row height', () => {
+      handsontable({
+        data: Handsontable.helper.createSpreadsheetData(5, 5),
+        rowHeaders: true,
+        manualRowResize: true,
+      });
+      const $rowHeaders = getInlineStartClone().find('tbody tr th');
+
+      expect($rowHeaders.eq(0).height()).forThemes(({ classic, main }) => {
```

Comment:
I think this can be simplified to something like this
```suggestion

      expect(getCell(0, -1).offsetHeight).forThemes(({ classic, main }) => {
```